docs/getting_started/videos/fast_data_no_code.mdx

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
can these pages get generated automatically from a json of videos? I think it lead to less maintenance 🤔